intl/msg_hash_us.h

```diff
@@ -209,11 +209,11 @@ MSG_HASH(
       )
 MSG_HASH(
       MENU_ENUM_LABEL_VALUE_AUTO_OVERRIDES_ENABLE,
-      "Load Override Files Automatically"
+      "Load Core Overrides Automatically"
```

Comment:
I wanted to mention that I removed "Configurations Per Core" label too. It might be a hidden or stray option under Settings. Either way, something got fixed in the new place so I removed it afterward. Just a heads up.